features-json/webxr.json

```diff
@@ -0,0 +1,368 @@
+{
+  "title":"WebXR Device API",
+  "description":"API for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays.",
```

Comment:
This definition ignores AR use cases which have applications outside of games. The spec was designed specifically (so it's author tells me) to address a spectrum of immersion so that a site could use as little or as much immersion as it wanted. See [my article](https://developers.google.com/web/updates/2018/05/welcome-to-immersive) for more on that.